packages/samples/headless-ssr/cypress/e2e/ssr.cy.ts

```diff
@@ -56,12 +56,29 @@ describe('headless ssr example', () => {
     cy.vitals(VITALS_THRESHOLD);
   });
 
```

Comment:
Can we add `firstInputSelector` to the vitals test above for the search box input and also add `fid`, `inp`?
https://www.npmjs.com/package/cypress-web-vitals

Reply:
Followed their docs and even split the `cy.vitals` call into `cy.startVitalsCapture` and `cy.reportVitals` like they do, but unfortunately, the console says `fid` and `inp` are ignored. Will merge as-is for now.

Reply:
Can you add a TODO with your findings.